src/utils/viewer/viewer.cpp

```diff
@@ -412,6 +432,24 @@ Viewer::Viewer(const Arguments& arguments)
   printHelpText();
 }  // end Viewer::Viewer
 
+void Viewer::switchCameraType() {
+  auto cameraSensor =
+      defaultAgent_->getSensorSuite().getSensors()["rgba_camera"];
+  auto cam = std::static_pointer_cast<esp::sensor::CameraSensor>(cameraSensor);
+
+  auto oldCameraType = cam->getCameraType();
+  switch (oldCameraType) {
+    case esp::sensor::SensorSubType::Pinhole: {
+      cam->setCameraType(esp::sensor::SensorSubType::Orthographic);
+      break;
+    }
+    case esp::sensor::SensorSubType::Orthographic: {
+      cam->setCameraType(esp::sensor::SensorSubType::Pinhole);
+      break;
+    }
+  }
```

Comment:
```suggestion
  switch (oldCameraType) {
    case esp::sensor::SensorSubType::Pinhole: {
      cam->setCameraType(esp::sensor::SensorSubType::Orthographic);
      return;
    }
    case esp::sensor::SensorSubType::Orthographic: {
      cam->setCameraType(esp::sensor::SensorSubType::Pinhole);
      return;
    }
  }

  CORRADE_INTERNAL_ASSERT_UNREACHABLE();
```

This way, when we get a fisheye camera next, this place won't silently stop working but yet (compared to putting that assert in a `default:` clause) the compiler will still notify you that you have an unhandled enum value when the `SensorSubType` gets extended.